Sources/Commands/SwiftTestTool.swift

```diff
@@ -277,7 +279,9 @@ public class SwiftTestTool: SwiftTool<TestToolOptions> {
                 toolchain: toolchain,
```

Comment:
we should remove sanitizers property from parallel test runner as well.

Reply:
done